src/less/icon-button/icon-button.less

```diff
@@ -19,15 +19,14 @@ a.icon-link {
     font-family: inherit;
     height: 40px;
     margin: 0;
-    min-width: 40px;
     padding: 0;
     vertical-align: text-bottom;
     width: 40px;
 
     > svg {
         .fill-token(icon-button-icon-foreground-color, color-foreground-primary);
 
-        max-width: 100%; // helps when width & height HTML attrs not set
```

Comment:
Why was this changed?

Reply:
I discussed this change in the notes for the PR, would like to hear your thoughts on it:

> Icons typically filled most of the button when a small modifier was applied, so I reduced `max-width` from `100%` to `75%`. Other values may be better (i.e. if we want to be mathematically precise and fill the circle with square icons it should be `70.71%`)

Reply:
Lets put it in. I think it's fine.